ci/jobs/test.yml

```diff
@@ -0,0 +1,14 @@
+jobs:
+- name: test
+  plan:
```

Comment:
Like with the other modules we need to put logic in here to ensure branches are in sync